test/test_gpu_tensorcore.cpp

```diff
@@ -4386,7 +4386,7 @@ TEST_F(NVFuserTest, FusionMatmulSchedulerEpilogueAlphaBetaBias_CUDA) {
 
   // NOTE: increasted absolute tolerance to silence false negative verification
   //       caused by different way of calculating reference
-  TORCH_CHECK(outputs[0].allclose(t8, 0.001, 0.001));
+  TORCH_CHECK(outputs[0].allclose(t8, 0.01, 0.01));
```

Comment:
Unrelated test fix

Reply:
added

Reply:
On what device have you seen this test failing due to too high relative error?

Reply:
Ok, good to know - I will make sure to run tests against ad102 for new features I add from now on to avoid issues like this.

Thanks!